src/particles/ImpactXParticleContainer.cpp

```diff
@@ -99,6 +99,12 @@ namespace impactx
         m_refpart = refpart;
     }
 
+    RefPart
+    ImpactXParticleContainer::GetRefParticle ()
```

Comment:
Upsi, I forgot that:
```suggestion
    ImpactXParticleContainer::GetRefParticle () const
```